contracts/cosmwasm-vm/cw-xcall-ibc-connection/src/contract.rs

```diff
@@ -152,7 +152,8 @@ impl<'a> CwIbcConnection<'a> {
             }
             #[cfg(not(feature = "native_ibc"))]
             ExecuteMsg::IbcChannelClose { msg } => {
-                self.ensure_ibc_handler(deps.as_ref().storage, info.sender)?;
+                self.ensure_ibc_handler(deps.as_ref().storage, info.clone().sender)?;
+                self.ensure_admin(deps.as_ref().storage, info.sender)?;
```

Comment:
It seems that this checks requires sender to be both ibc handler as well as admin?